packages/widget/src/components/MainButton.tsx

```diff
@@ -14,6 +16,13 @@ export type MainButtonProps = {
   loading?: boolean;
   loadingTimeString?: string;
   onClick?: () => void;
+  extra?: ReactNode;
+  route?: RouteResponse
+  useBrandColorForLoadingAnimation?: boolean;
```

Comment:
Because currently this will used only for go fast I suggest to change the prop name
```suggestion
  isGoFast?: boolean;
```